Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchRemoteDebugJob.java

```diff
@@ -485,7 +485,9 @@ static SparkBatchRemoteDebugJob factory(
         driverOption = (carriedDriverOption.trim() + " " + driverDebugOption).trim();
         HashMap<String, Object> jobConfigWithDebug = new HashMap<>(submissionParameter.getJobConfig());
         SparkConfigures sparkConfigWithDebug = new SparkConfigures(sparkConf);
+
         sparkConfigWithDebug.put(sparkJobDriverJvmOptionConfKey, driverOption);
+        sparkConfigWithDebug.put("spark.yarn.maxAppAttempts", "1");
```

Comment:
I think we should have a way to make it visual for customers (on the debug panel?).  And let check the value is set or not first by customers's configuration file.